source/info-tool/ev3dev-system-info.sh

```diff
@@ -32,6 +36,14 @@ print_val() {
   printf "%-20s%s\n" "$1: " "$2"
 }
 
+print_fence_if_markdown() {
+  if [ "$FORMAT_MARKDOWN" = true ]; then
+    echo '```'
+  fi
+}
+
+print_fence_if_markdown
```

Comment:
I also liked your idea of printing something like "Output of ev3dev-sysinfo:" somewhere around here.